### PR TITLE
Cleanup of assertions page

### LINKED ISSUE #106: "Overload:" artifact in "Assertions" article
https://msdn.microsoft.com/en-us/visualfsharpdocs/conceptual/assertions-[fsharp]

image

I'm not sure, but it seems that `Overload:System.Diagnostics.Debug.Assert` should be a link.

### PR DESCRIPTION
**Pull request by ReedCopsey**
Fixes #106

### CODE REVIEW

File: docs/conceptual/assertions-[fsharp].md [hunk 1]
```diff
@@ -30,14 +30,14 @@ Assertion checking is enabled only when you compile in Debug mode; that is, if t
 
 The assertion failure error cannot be caught by using F# exception handling.
 
-
->[!NOTE] {The **assert** function resolves to **System.Diagnostics.Debug.Assert**. For more information, see **Overload:System.Diagnostics.Debug.Assert**.
-
+>[!NOTE] {The **assert** function resolves to **System.Diagnostics.Debug.Assert**. For more information, see **System.Diagnostics.Debug.Assert**.
```
Comment (dend) [inline docs/conceptual/assertions-[fsharp].md]: You can also remove the brackets here because those are unnecessary.

Comment (ReedCopsey) [inline docs/conceptual/assertions-[fsharp].md]: @dend So this can be `!NOTE { ...` ?

Comment (dend) [inline docs/conceptual/assertions-[fsharp].md]: Sorry, `[!NOTE] Some text here`

Comment (ReedCopsey) [inline docs/conceptual/assertions-[fsharp].md]: Ahh, okay- remove braces ;)

Comment (ReedCopsey) [inline docs/conceptual/assertions-[fsharp].md]: @dend Please check now

**Comment by dend** [comment]
@ReedCopsey ah, braces 👍 That's what I get for being a non-native English speaker lol

**Pull request closed without merging**